django-stubs/forms/forms.pyi

```diff
@@ -29,6 +29,7 @@ class BaseForm:
     fields: Dict[str, Any] = ...
     renderer: BaseRenderer = ...
     cleaned_data: Any = ...
+    changed_data: List[str]: ...
```

Comment:
Can we make it a `@propery`? I guess it is the closest match.

Reply:
Oh good idea, I changed it 👍